src/zalopay/models/models.ts

```diff
@@ -41,6 +44,12 @@ export * from "./refundCreateRequest";
 export * from "./refundCreateResponse";
 export * from "./refundQueryRequest";
 export * from "./refundQueryResponse";
+export * from "./zODCreateInvoiceRequest";
```

Comment:
zODCreateInvoiceRequest -> it should be zodCreateInvoiceRequest

Reply:
It is generated, but the class name is still ok (user experience is also ok) so I ignore this.